lib/suspenders/app_builder.rb

```diff
@@ -366,6 +372,14 @@ def create_heroku_apps(flags)
       create_production_heroku_app(flags)
     end
 
+    def create_heroku_pipeline
+      pipeline_name = app_name.dasherize
+
+      run_heroku "pipelines:create -a #{app_name.dasherize}-staging", "staging"
```

Comment:
When writing this we were assuming it would not work without the `-staging` suffix. Was that assumption wrong?